src/ztc/api/serializers/eigenschap.py

```diff
@@ -50,6 +51,8 @@ class EigenschapSerializer(serializers.HyperlinkedModelSerializer):
         source="zaaktype", read_only=True, slug_field="identificatie"
     )
 
+    catalogus = serializers.SerializerMethodField()
```

Comment:
```suggestion
    catalogus = serializers.HyperlinkedRelatedField(source="zaaktype.catalogus", read_only=True)
```
should work and produce better API schema generation